docs/protocol/network/data-structures/transaction.md

```diff
@@ -0,0 +1,136 @@
+# Transaction
+
+In the Witnet network protocol, a `transaction` is formatted as follows:
+
+| Field     |    Type    | Description                                                               |
+| --------- | :--------: | ------------------------------------------------------------------------- |
+| `version` |   `u32`    | The transaction data format version number                                |
+| `inputs`  | `[input]`  | A list of 1 or more transaction inputs                                    |
+| `outputs` | `[output]` | A list of transaction outputs                                             |
+| `claims`  | `[claim]`  | A list of 1 or more claims (i.e. spending conditions), one for each input |
```

Comment:
The requirement for _1 or more_ inputs and claims is in conflict with the existence of coinbase transactions.

Reply:
Then, it should be `1 or more transaction outputs` to enforce at least the coinbase. :)

Reply:
Only implication then is that you can't produce transactions that donate all their value to the miner without changing or burning at least one _satowit_. I'm totally ok with that.

Reply:
Is it OK as follows?

```table
| Field     |    Type    | Description                                 |
| --------- | :--------: | ------------------------------------------- |
| `version` |   `u32`    | The transaction data format version number  |
| `inputs`  | `[input]`  | A list of transaction inputs                |
| `outputs` | `[output]` | A list of 1 or more transaction outputs     |
| `claims`  | `[claim]`  | A list of claims (i.e. spending conditions) |
```

Inputs and claims may be empty due to the `coinbase` transaction. The claim for mining the block is included in the block itself as leadership proof.